packages/dds/tree/src/test/shared-tree-core/sharedTreeCore.spec.ts

```diff
@@ -195,27 +194,6 @@ describe("SharedTreeCore", () => {
 		assert.equal(getTrunkLength(tree), 6 - 3);
 	});
 
-	it("can complete a transaction that spans trunk eviction", () => {
```

Comment:
Seems like a good thing to keep coverage for.

Reply:
As discussed offline, it's okay to drop this because it is not correct (or useful that we can tell)